index.js

```diff
@@ -23,24 +24,19 @@ app.use(
 
 app.use(cookieParser());
 app.use(urlencoded({ extended: false }));
-app.use(Routes);
 
-app.use((error, request, response, next) => {
-  response.status(error.status || 500).json({
-    message: error.message || 'Oops! something went wrong',
-    code: error.code || 500,
-    data: error,
-  });
-  next();
-});
+app.get('/', (req, res) => res.status(200)
+  .send('<h1>Welcome to Edustripe authentication service!</h1>'));
+
+app.use('/api/v1', Routes);
 
-app.use('*', (request, response) => {
-  response.status(404).send({
-    message: 'Requested resource not found!',
-    code: 404,
-  });
+app.use((error, req, res, next) => {
+  if (res.headersSent) return next(error);
+  return response.error(res, error.status || 500, error, error.message || 'Internal Server Error');
 });
 
+app.use('*', (req, res) => response.notFound(res));
+
 app.use(helmet());
```

Comment:
Great work here Samuel, however, can we use helmet as the first middleware? It helps add the headers earlier in the application

Reply:
Done